spec/unit/action/deploy/module_spec.rb

```diff
@@ -103,21 +103,14 @@
       end
 
       before do
-        @modules = []
         allow(subject).to receive(:visit_environment).and_wrap_original do |original, environment, &block|
-          mod = R10K::Module::Local.new(environment.name, '/fakedir', {}, environment)
-          if mod.name == 'first'
-            expect(environment).to receive(:generate_types!)
+          if environment.name == 'first'
+            expect(environment).to receive(:deploy).and_return(['first'])
+            expect(environment).to recieve(:generate_types!)
```

Comment:
Adding this expectation causes some kind of error that makes rspec bail in such a way that `environment.deploy` never gets called. I can't figure out why. Wrapping these exceptions in a `begin...rescue` makes it go away, but also doesn't catch any exception.

Reply:
Typo, derp. Thanks Molly!